commerce/domain/pricing/service/classes/PricingServiceSample.cls

```diff
@@ -141,6 +145,49 @@ public class PricingServiceSample extends commercestorepricing.PricingService {
     return txnResponse;
   }
 
+  private Map<String, PriceDataFromExternalService> getPriceFromStaticResponse(Set<String> skus) {
+        String responseJson = '{';
+        for(String sku : skus) {
+            Double price = 0.00;
+            if (sku == 'SKU_FOR_TEST') {
```

Comment:
what does this SKU_FOR_TEST represent? 

the list of productIds that this passed in as input will never be equal to SKU_FOR_TEST

Reply:
The reason I am asking is because price = 100 never takes into effect since none of the productIds = SKU_FOR_TEST

Reply:
I agree, we should get rid of `SKU_FOR_TEST`..This came from here - https://github.com/gurpreetsainisalesforce/commerce-on-lightning/blob/571fd7a5ace6b18109c24bd8a294d8dde2da4e80/examples/b2b/checkout/integrations/classes/B2BPricingSample.cls
Will verify if this is needed or not